docs/source/dev_guide/review_process.rst

```diff
@@ -0,0 +1,95 @@
+.. _review_process:
+
+.. contents::
+   :local:
+
+Review process
+--------------
+
+This short description should serve as a rough guideline on how we deal with
+reviews on pull requests (PR). Some PRs are easy, others are hard. Some require
+multiple changes and continuous feedback, others can be merged in
+flawlessly. The purpose of this text is to communicate our intent and style and
+should not be seen as *law*.
+
+After successfully adding a new feature, doing maintenance, sorting out a bug or
+adding new documentation, in short: contributing to the codebase in any form,
+you should ensure that all existing and new tests pass. These include tests for
+functionality on the C++ as well as the python side but also include tests for
+code style as far as they can be automated (see our :ref:`Coding convention
+<coding-convention>` for details on the style). We also have a custom ``make``
+target to help with autoformatting C++ as well as Python code.
+
+Further, we provide a PR template on Github with several checkmarks to help
+start and streamline the review process. Please make use of it, although you can
+remove points that are irrelevant to the PR. This template is intended to help
+you in preparing your contribution for the review process and includes the main
+points most commonly raised by reviewers. Also we highly encourage the use of
+the *Draft* PR functionality Github provides. We see it as a way to start a
+discussion about the contribution and the proposed changes and it does not have
+to be polished at that stage. The difference between a Draft PR and a normal PR
+is minor. The Draft stage signals that your contribution is not ready for review
+but up for discussion. But it is nevertheless treated by like a normal PR in the
+sense that all the continuous integration tools are applied and there is an
+immediate feedback from automatic tests. Do not hesitate to contact us at any
+stage if you are unsure about anything.
+
+Once you have added something to the code base, please check locally if the
+tests pass. These tests include the code tests as well as the tests for code
+style as far as they can be checked automatically. You should also add tests for
+new functionality. Likewise, if your changes break existing tests, be sure to
```

Comment:
```suggestion
new functionalities. Likewise, if your changes break existing tests, be sure to
```

Reply:
Actually, the previous one was correct; "functionality" can't be pluralized in this context.